hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -733,6 +733,11 @@ update ::
   Event tx ->
   Outcome tx
 update Environment{party, signingKey, otherParties} ledger st ev = case (st, ev) of
+  (IdleState, ClientEvent (ModifyPeers peers)) ->
+    OnlyEffects
+      [ ClientEffect $ PeersModified peers
+      , NetworkEffect $ PeersUpdated peers
```

Comment:
Why the network effect 🤔?

Reply:
NetworkEffect is used to trigger the reconnection
I thought it was best to split the server output and the network msg.
Beware this is the only msg which is not broadcast to rest of peers.
Please check `Node.processEffect` to see if u like the approach or not.
Looking forward to your comments.

Reply:
I see!